sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -316,6 +316,7 @@ public partial class TableSasBuilder
         public override bool Equals(object obj) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override int GetHashCode() { throw null; }
+        public static Azure.Data.Tables.Sas.TableSasBuilder Parse(System.Uri uri) { throw null; }
```

Comment:
Why not ctor?

Reply:
I went back and forth about this. Perhaps the ctor makes more sense here, but I'm not sure.

Reply:
I think TableUriBuilder and TableSasBuilder should be consistent one way or another.